arduino-ide-extension/src/browser/arduino-ide-frontend-module.ts

```diff
@@ -453,7 +453,10 @@ export default new ContainerModule((bind, unbind, isBound, rebind) => {
   bind(BoardsConfigDialogWidget).toSelf().inSingletonScope();
   bind(BoardsConfigDialog).toSelf().inSingletonScope();
   bind(BoardsConfigDialogProps).toConstantValue({
-    title: nls.localize('arduino/common/selectBoard', 'Select Board'),
+    title: nls.localize(
+      'arduino/board/boardConfigDialogTitle',
+      'Select other board and port'
```

Comment:
```suggestion
      'Select Other Board and Port'
```

Unfortunately we not so organized as to have a style guide for such things, so the established conventions must serve in its place.

I did a survey of the dialog titles and it seems like the prevailing style for dialog titles is https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style:

## Title case

- Additional Boards Manager URLs
- WiFi101 / WiFiNINA Firmware Updater
- Upload SSL Root Certificates
- Add File
- Board Info
- Software Update

CC: @91volt 

## Sentence case

- Save sketch folder as...
- Select a zip file containing the library you'd like to add

## Inconclusive

- Open
- Preferences
- Arduino IDE
  (the "About" dialog)

Reply:
Actually this change wasn't intentional. I'm changing it back, thanks for spotting it @per1234 ! (and sorry for the late reply 🙏)

Reply:
EDIT: I mean, the intention was to change the title from "Select Board" to  "Select Other Board and Port" with the case you suggested.

Reply:
I noticed the reworking you did of that dialog title and I think it is a very nice improvement even beyond the improved support for small window dimensions!

I end up having to describe the UI elements of the IDE in text quite often, which makes me especially conscious of the text labels such as dialog titles/headings.